open-api-spec.yaml

```diff
@@ -1370,106 +1388,6 @@ components:
               updatedAt: '2022-04-01T10:24:36.11Z'
               description: 'Hi Mo, this is for the cappuccino I bought for you the other day.'
               externalRef: Coffee w/ Mo on 10 March 22
-    list-outgoing-payments:
```

Comment:
Why is this removed but we still have `list-incoming-payments`?

Reply:
For some reason I was thinking `list-outgoing-payments` had a separate incoming payment schema without the stream credentials,
It _seems_ like `list-incoming-payments` could also be removed.

Reply:
The only reason to keep it is if we want to enforce that STREAM credentials are never part of the list response, not even the connection url.

Reply:
But then we shouldn't reference the incoming payment model

Reply:
PR: #137